src/components/RadioGroupList/RadioGroupList.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from "react";
+
+import { tv } from "tailwind-variants";
+
+import { cn } from "@/lib/utils";
+import { RadioGroup, RadioGroupItem } from "@/primitives/RadioGroup";
+
+const radioGroupList = tv({
+  slots: {
+    root: "flex flex-col",
+    header: "mb-4 font-sans text-[16px] font-bold leading-[24px]",
```

Comment:
^ Just curious , is font-sans unique to just this component ?  I assumed it would be consistent across the library